actor.js

```diff
@@ -1,44 +1,62 @@
 const Message = require('primea-message')
-const PortManager = require('./portManager.js')
-const DeleteMessage = require('./deleteMessage')
+const CapsManager = require('./capsManager.js')
+const Inbox = require('./inbox.js')
 
-module.exports = class Kernel {
```

Comment:
Now that this is called actor instead of kernel, should we update docs/any var names for consistency?

Reply:
yes, defaintly